### PR TITLE
Fix invalid less break point

### PR DESCRIPTION
**Pull request by mrtuvn**
## Purpose of this pull request

This pull request (PR) update section `CSS in responsive design`. Add more example and also fix the incorrect part in example 

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_css.html
https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/responsive-web-design/rwd_css.html

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_css.md [hunk 1]
```diff
@@ -85,14 +85,20 @@ For grouping style rules in certain media queries the `.media-width()` mixin use
...
 //  Tablet
 //  _____________________________________________
 
 // This will add styles for tablet devices. When using native media-queries, we recommend wrapping your media-queries with media-width Magento mixins or media-target
 & when (@media-target = 'desktop'), (@media-target = 'all') {
-    @media only screen and (min-width: @screen__m) and (max-width: @screen__xl - 1) {
-        // your code
+    @media only screen and (min-width: @screen__m) and (max-width: (@screen__xl - 1)) {
+        // styles for breakpoint >= 768px and < 1440px
     }
```
Comment (mrtuvn) [inline src/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_css.md:102]: add missing round brackets

**Review by BarnyShergold — APPROVED** [review]

**Comment by mrtuvn** [comment]
squash commits to one

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**